### ISSUE TITLE
Sanitize Message Delivery

### ISSUE THREAD

**Issue by JustRevDice** [Kaz040/socket-programming-for-DAS#1]
Inputting text including the delimiter exhibits potentially breaking behavior.

It's solvable by choosing a different delimiter going by likeliness of usage, but current implementation allows for sanitizing sent data.

**Comment by Kaz040** [comment]
That's true. I believe "|" is one delimiter that is likely used... maybe ^,´ or ~. Thought I tough of something else that would make the complete message readable even if it contains the delimiter. I will implement that this weekend. I am not sure if it is the best implementation idea but I will look more into it in coming days.

**Comment by JustRevDice** [comment]
I'll look into it once I have time and open a PR if I find a better solution. Though appending anything post data[3] is reasonable enough.
